src/basics.md

```diff
@@ -1385,6 +1386,52 @@ fn main() {
 }
 ```
 
+[ex-examine-date-and-time]: #ex-examine-date-and-time
+<a name="ex-examine-date-and-time"></a>
+## Examine the date and time
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Gets the current UTC [`DateTime`] and its hour/minute/second via [`Timelike`]
+and its year/month/day/weekday via [`Datelike`].
+
+```rust
+extern crate chrono;
+use chrono::{Datelike, Timelike, Utc};
+
+fn main() {
+    let now = Utc::now();
+
+    // available due to chrono::Timelike
+    let (is_pm, hour) = now.hour12();
+    println!(
+        "The current UTC time is {:02}:{:02}:{:02} {}",
+        hour,
+        now.minute(),
+        now.second(),
+        if is_pm { "PM" } else { "AM" }
+    );
+    println!(
+        "And there have been {} seconds since midnight",
+        now.num_seconds_from_midnight()
+    );
+
+    // available due to chrono::Datelike
```

Comment:
The comment repeats what is said in recipe description (few lines above). Does it make sense to remove it (to make example slightly shorter)?

Reply:
Yeah, this is a good idea. 
Lets drop the comments and we are ready to merge!

Also, the example title is still a little off but I can't come up with a better one at the moment. @CrockAgile If you can come up with something slightly more natural please propose one but it will not block the PR. 
Well done!

Reply:
@budziq I also can't get a better title in my head, but I agree it is lacking. "Get current date and time"? I am open to suggestions. Hard to be more detailed/exciting when there are already examples for parsing and modifying datetimes.